### ISSUE TITLE
Update version of marked used?

### ISSUE THREAD

**Issue by jthomas** [nimbella/commander-cli#40]
When I'm installing this dependency for this Nimbella CLI - it's complaining about the following:

```
 ~/code/nimbella/nimbella-cli  master ⇡2  npm install                                                                                                            1 ✘  10:50:12
npm WARN ERESOLVE overriding peer dependency
npm WARN Found: marked@0.8.2
npm WARN node_modules/marked
npm WARN   marked@"^0.8.2" from commander@0.1.0
npm WARN   node_modules/commander
npm WARN     commander@"nimbella/commander-cli#9bde76cc8d9e80ebda444f2d5a10102dc1590aca" from the root project
npm WARN
npm WARN Could not resolve dependency:
npm WARN peer marked@"^1.0.0 || ^2.0.0" from marked-terminal@4.1.1
npm WARN node_modules/marked-terminal
npm WARN   marked-terminal@"^4.0.0" from commander@0.1.0
npm WARN   node_modules/commander
```

marked is now at 2.0.3 - whereas this repo is using a pre-1.0.0 version. Could we upgrade that version? Happy to sumbit a PR....

**Comment by joshuaauerbachwatson** [comment]
It's ok with me and it would be great to continue rolling forward with the stable version release so go ahead and make this PR.   I may hold off on merging it (and we might need to pause the release) to consult @niks3089 because he is also making dependency changes in commander.

**Comment by niks3089** [comment]
I am ok with this change as well. Thanks James

**Comment by niks3089** [comment]
fixed by #41
